hoot-core/src/main/cpp/hoot/core/visitors/RemoveInvalidMultilineStringMembersVisitor.cpp

```diff
@@ -94,10 +94,20 @@ void RemoveInvalidMultilineStringMembersVisitor::visit(const ElementPtr &e)
                   rev->addElement(MetadataTags::RoleReviewee(), id2);
               }
             }
-            rev->removeElement(r->getElementId());
+            //  There is a one-to-one relation here, replace the multilinestring with the only way
+            if (rev->getMembers().size() == 1 && r->getMembers().size() == 1)
+              rev->addElement(MetadataTags::RoleReviewee(), r->getMembers()[0].getElementId());
+            //  Don't remove multilinestring relations that are members of a review relation
+            //  only contains one member that is the original multilinestring
```

Comment:
So what is happening here is that a multilinestring relation in the source data will get wrapped in a hoot bookkeeping relation if it ends up being involved in a review?
So that situation creates a bookkeeping relation with only one member (the multilinestring relation present in the source data)?
I thought the code would only decompose relations one level deep.